aspnetcore/release-notes/aspnetcore-9/samples/SignalRChatTraceExample/Program.cs

```diff
@@ -0,0 +1,52 @@
+// <snippet_trace_signalr_server>
+using OpenTelemetry.Trace;
+using SignalRChat.Hubs;
+
+// Set OTEL_EXPORTER_OTLP_ENDPOINT environment variable depending on where your OTEL endpoint is.
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddRazorPages();
+builder.Services.AddSignalR();
+
+builder.Services.AddRazorPages();
```

Comment:
Remove duplicate code

Reply:
Thanks for the catch, Brennan, sorry I missed that.  Correcting....